### ISSUE TITLE
Teamblackminer latest releases won't launch on MSOS (Minerstat OS)

### ISSUE THREAD

**Issue by Gloom536** [sp-hash/TeamBlackMiner#401]
I'm using MSOS which is linux based, and for some reason if I try to launch latest release, and the last few lastest update, teamblack miner won't launch at all.

I tried randomly launching 1.74 and it works. Something must be wrong in latest release.

**Comment by sp-hash** [comment]
Did you try 2.07?

**Comment by Gloom536** [comment]
No, I tried 2.06, 2.07 just got released from what I see, but not available yet on Minerstat, I will try when they update it.

But from what you wrote on patch note, it may not help me, I'm mining VTC, not ETC/ETH

**Comment by sp-hash** [comment]
What error message are you getting when you launch the miner?

**Comment by Gloom536** [comment]
![image](https://github.com/sp-hash/TeamBlackMiner/assets/105163385/9231cba7-3b44-4119-ac32-88151a799eb3)
On 2.06 I don't see anything, but on "older" release I see this.

I'm testing at this moment what is the latest release that work, I will let you know.

**Comment by sp-hash** [comment]
You need a driver that can run cuda 12 code

**Comment by Gloom536** [comment]
Actually I'm on Nvidia 525.105.17

I have access to those driver, which one I should use? 
<img width="184" alt="image" src="https://github.com/sp-hash/TeamBlackMiner/assets/105163385/c6a00c5a-c9eb-4d8b-831e-7105139bca04">

**Comment by sp-hash** [comment]
you need a driver newer than 525.60.11 so select the highest number. 535.113.01

**Comment by Gloom536** [comment]
It didn't fi this issue, even that with this driver other miner stopped working too.

Minerstat dev tell me it's because your newer miner release doesn,t support Linux 18.04. Why did you stop supporting this widespreaded version?

**Comment by sp-hash** [comment]
But ubuntu 18.04 is not supported anymore. Minerstat should upgrade to 20.04.

https://techcommunity.microsoft.com/t5/linux-and-open-source-blog/canonical-ubuntu-18-04-lts-reaching-end-of-standard-support/ba-p/3822623

**Comment by sp-hash** [comment]
The 20.04 build should work on 18.04 with some libs upgraded. What is the error message you are getting with the latest nvidia driver?

**Comment by Gloom536** [comment]
You are the only miner that doesnt work with minerstat (that I know of) because of linux 18.04, no other miner "force" to upgrade to newer linux version. 

With the latest driver I do not see any error, it just crash in loop.

I dont know how I can upgrade the libs.

**Comment by Gloom536** [comment]
thank you, I'll ask Minerstat dev to upload this version to see if it works!
